backend/app/controllers/api/v1/instructor/instructors_controller.rb

```diff
@@ -13,10 +13,48 @@ def index
 
         # Find the IDs of all instructors that are associated with the same positions we are
         fellow_instructor_ids =
-            active_instructor.positions.joins(:instructors).pluck(
-                :"instructors.id"
-            )
+            active_instructor
+                .positions
+                .joins(:instructors)
+                .pluck(:"instructors.id")
 
         render_success Instructor.order(:utorid).find(fellow_instructor_ids)
     end
+
+    # POST /instructors
+    def create
+        active_user = ActiveUserService.active_user request
+
+        find_instructor
+
+        # We are only allowed to update information about ourselves, which is determined
+        # by the active user having the same utorid as the instructor to be updated.
+        unless @instructor.utorid == active_user.utorid
+            render_error(
+                message:
+                    "Insufficient permissions to update instructor with utorid='#{
+                        @instructor.utorid
+                    }'"
+            )
+        end
+
+        render_on_condition(
+            object: @instructor,
+            condition: proc { @instructor.update!(instructor_params) }
+        )
+    end
+
+    private
+
+    # Find an instructor looking up by `id` first and then by `utorid`
```

Comment:
If we use utorid as a primary key and it is required by the route, why would we want to also query by id?
To think the other way around, in our last meeting we talked about how having an id implies it is an update, then why would we want to also update by utorid when id is not given? Is it a special case?

Reply:
It is possible to change an instructor's UTORid